.github/workflows/main.yml

```diff
@@ -1,24 +1,24 @@
-name: Check Config JSON file keys
-
-on:
-  push:
-    paths:
-      - 'config/**/*.json'
-  pull_request:
-    paths:
-      - 'config/**/*.json'
-
-jobs:
-  check-keys:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      - name: Install jq
-        run: sudo apt-get update && sudo apt-get install -y jq
-
-      - name: Check JSON keys
-        run: |
-          bash ./.github/workflows/check-json.sh
-
+name: Check Config JSON file keys
+
+on:
+  push:
+    paths:
+      - 'config/**/*.json'
+  pull_request:
+    paths:
+      - 'config/**/*.json'
+
+jobs:
+  check-keys:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install jq
+        run: sudo apt-get update && sudo apt-get install -y jq
+
+      - name: Check JSON keys
+        run: |
+          bash ./.github/workflows/check-json.sh
+          
```

Comment:
still missing trailing new line. Please configure your editor to automatically add it.